Containerfile

```diff
@@ -1,28 +1,24 @@
-FROM registry.access.redhat.com/ubi7/ubi AS builder
-ENV SEARCHD_RPM=sphinx-2.2.11-1.rhel7.x86_64.rpm \
-    SEARCHD_RPM_SHA=959b04eb3f7fb2314d7a2702b61e9b3e627b66b1a8574dece21c0592be1b90e2 \
-    SEARCHD_SRC=sphinx-2.2.11-release.tar.gz \
-    SEARCHD_SRC_DIR=sphinx-2.2.11-release \
-    SEARCHD_SRC_SHA=6662039f093314f896950519fa781bc87610f926f64b3d349229002f06ac41a9
+FROM quay.io/centos/centos:stream9 AS builder
+ENV SEARCHD_REF=6.2.12 \
+    SEARCHD_REPO=https://github.com/manticoresoftware/manticoresearch.git \
+    CC=clang-16 \
+    CXX=clang++-16 \
+    BUILD_PATH=/tmp/manticore_uselessly_long_path_to_prevent_rpm_build_issues \
```

Comment:
what is this? :sweat_smile:

Reply:
RPMs (the debug ones IIRC) fail to be generated when this path is short. You can try :)
Not pretty - yes.

Reply:
Funnily see how it is done in manticore docs they used `/manticore_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa`

https://manual.manticoresearch.com/Installation/Compiling_from_sources#Configuration-flags-and-options

Reply:
Yes, I noticed that later :upside_down_face: 
It is crazy though... Normally the opposite happens - stuff breaks when the resulting path is too long... :zany_face: